### PR TITLE
Rancherize + Create/Seed 'actual' DynamoDB table(s)

### PR DESCRIPTION
**Pull request by waysidekoi**
This PR allows you to: `mdx-service deploy --env=<ENV>` to deploy this application to a Rancher environment. It will only work if the `DB_ENDPOINT` secret is set for the given environment: https://github.com/mdx-dev/wave-machine/compare/feature/rancherize?expand=1#diff-a6f3209d4249f5c291b61e21626c86d4R6, currently only QA. But, once it's deployed, you can execute: `rake db:seed` to get some dummy data into the DynamoDB tables and see the data for yourself.

Likewise, `rake db:drop`, despite being a misnomer, will _only_ destroy DynamoDB tables belonging to the application. This also reinforced by the IAM policy created here: https://github.com/mdx-dev/wave-machine/compare/feature/rancherize?expand=1#diff-7da79f5b52019d5091fb01d08ac3d518

### CODE REVIEW

File: spec/factories/user.rb [hunk 1]
```diff
@@ -4,12 +4,5 @@
     email              'bob@usermail.com'
     date_of_birth      { 25.years.ago }
     gender             'male'
-    phone_number       '123 456 7890'
-    insurance_company  'Big Old Insurance Company'
-    plan_type          'ppo'
-    policy_name        'some policy'
-    policy_holder_name 'Bob User'
-    group_number       1
-    policy_number      23
```
Comment (rall) [inline spec/factories/user.rb]: we want to keep this lot please!

File: lib/tasks/seed.rake [hunk 2]
```diff
@@ -0,0 +1,42 @@
+namespace :db do
```
Comment (rall) [inline lib/tasks/seed.rake]: should this file be named db.rake now? or dynamodb.rake?

Comment (waysidekoi) [inline lib/tasks/seed.rake]: Yup, that's better. I started with the intention of "seeding" and never renamed as it changed

File: spec/factories/user.rb (hunk 1 above)
Comment (waysidekoi) [inline spec/factories/user.rb]: Reverted! I must have hit a diff kind of snag to warrant the deletion, thinking these were the cause

**Comment by waysidekoi** [comment]
Added `phone_number` back

**Comment by rall** [comment]
👍

**Pull request merged**